pkg/backup/backup/backup_tracker.go

```diff
@@ -53,11 +55,31 @@ type trackDepends struct {
 
 // NewBackupTracker returns a BackupTracker
 func NewBackupTracker(deps *controller.Dependencies, statusUpdater controller.BackupConditionUpdaterInterface) BackupTracker {
-	return &backupTracker{
+	tracker := &backupTracker{
 		deps:          deps,
 		statusUpdater: statusUpdater,
 		logBackups:    make(map[string]*trackDepends),
 	}
+	go tracker.initTrackLogBackupsProgress()
+	return tracker
+}
+
+// initTrackLogBackupsProgress lists all log backups and track their progress.
+func (bt *backupTracker) initTrackLogBackupsProgress() {
+	backups, err := bt.deps.Clientset.PingcapV1alpha1().Backups("").List(context.TODO(), metav1.ListOptions{})
```

Comment:
Can you add retry here? if it failed, it will be a critical error, not just a warning.

Reply:
yes, i can, but it may be not a critical error, because if the log backup start as normal, it can be added to tracker.